airbyte-integrations/connectors/source-github/source_github/schemas/issues.json

```diff
@@ -164,9 +165,11 @@
         },
         "patch_url": {
           "type": ["null", "string"]
+        },
+        "merged_at": {
+          "type": ["null", "string"]
         }
```

Comment:
Same - type": "string" + "format": "date-time"

Reply:
it can be incompatible schema change database `string` type -> `datetime` time
we need to decide when it's good time to do this change

Reply:
I think this should be a "format": "date-time" - all the properties that end in `_at` are timestamps